app/cdap/components/shared/ConfigurationGroup/index.tsx

```diff
@@ -161,9 +161,7 @@ const ConfigurationGroupView: React.FC<IConfigurationGroupProps> = ({
 
   function handleValueChanges(changedValues, params: { [key: string]: boolean } = {}) {
     let fcg = filteredConfigurationGroups;
-    if (params.updateFilteredConfigurationGroups) {
-      fcg = filterByCondition(configurationGroups, widgetJson, pluginProperties, changedValues);
-    }
+    fcg = filterByCondition(configurationGroups, widgetJson, pluginProperties, changedValues);
```

Comment:
Do we have any cases where `updateFilteredConfigurationGroups` should be false? Or, would it be better to add `updateFilteredConfigurationGroups: true` in the calling function for this scenario?

Reply:
I found one usage of setting it to true. But I believe this should apply to many components like toggle, radiogroup, select, etc., since they usually change what to show. Then is it really necessary to differentiate the cases? The only downside is one extra function call for all value changes. Let me know what you think

Reply:
I pushed some new changes that use this flag

Reply:
Thanks. I don't remember the exact path, but I think we had concerns about backwards compatibility when we added that functionality.